hyper_click_command.py

```diff
@@ -71,7 +71,13 @@ def is_valid_line(self, line_content):
 
     def is_visible(self, event=None):
         view = self.view
+
+        # old style discouraged single-string selector
         selector = self.settings.get('selector')
+        if not selector:
+            # new style list of strings selectors
+            selector = "(" + ")|(".join(self.settings.get('selectors')) + ")"
```

Comment:
I would leave the `( ... )` braces up to each selector, they're only needed in specific cases. Also, AFAIK an `OR` in a selector is like CSS: `,` not `|`.

Reply:
> I would leave the ( ... ) braces up to each selector, they're only needed in specific cases. 

Not sure what you mean.

---

> Also, AFAIK an OR in a selector is like CSS: , not |.

https://www.sublimetext.com/docs/selectors.html#logical_operators

> The logical OR operator is | or ,. If either the selector to the right or left of the operator is matched, the expression will be a match.

Reply:
Ha, learned something new. 👍 This is excellent then.